scripts/jenkins/jobs-ibs/openstack-mkcloud.yaml

```diff
@@ -0,0 +1,489 @@
+- job:
+    name: openstack-mkcloud
+    node: openstack-mkcloud
+    description: |
+     "!!! This job is automatically pushed from GIT to ci.suse.de !!!
+      REMEMBER: changes to this job will be overwritten after a few hours - make sure to create a PR for your changes
+      This job runs https://github.com/SUSE-Cloud/automation/blob/master/scripts/mkcloud results are 
+      analyzed by https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/mkcloudresultcheck.pl
+      on mkcloudresultcheck@mkcloudresultcheck.cloud.suse.de every 15 minutes"
+
+    logrotate:
+        numToKeep: 100
+        daysToKeep: 300
+
+    properties:
+      - authorization:
+          cloud:
+               - job-build
+               - job-cancel
+               - job-configure
+               - job-delete
+               - job-discover
+               - job-read
+               - job-status
```

Comment:
its in https://ci.suse.de/job/openstack-mkcloud/configure Enable project-based security section.